docs/data/material/getting-started/templates/checkout/PaymentForm.tsx

```diff
@@ -1,65 +1,250 @@
-          <FormControlLabel
-            control={<Checkbox color="secondary" name="saveCard" value="yes" />}
-            label="Remember credit card details for next time"
-          />
-        </Grid>
-      </Grid>
-    </React.Fragment>
+    <Stack spacing={3}>
+      <FormControl component="fieldset" fullWidth>
+        <RadioGroup
+          aria-label="paymentType"
+          name="paymentType"
+          value={paymentType}
+          onChange={handlePaymentTypeChange}
+          sx={{
+            display: 'flex',
+            flexDirection: { sm: 'column', md: 'row' },
+            justifyContent: 'center',
+            gap: { sm: '', md: 2 },
+          }}
+        >
+          <Card
+            raised={paymentType === 'creditCard'}
+            sx={{
+              maxWidth: { sm: '100%', md: '50%' },
+              mb: 2,
+              flexGrow: 1,
+              outline: '1px solid',
+              outlineColor:
+                paymentType === 'creditCard' ? 'primary.main' : 'divider',
+              backgroundColor:
+                paymentType === 'creditCard' ? 'background.default' : '',
+            }}
+          >
+            <CardActionArea onClick={() => setPaymentType('creditCard')}>
+              <CardContent sx={{ display: 'flex', alignItems: 'center' }}>
+                <CreditCardRoundedIcon color="action" sx={{ marginRight: 2 }} />
+                <Typography variant="subtitle1">Card</Typography>
+              </CardContent>
+            </CardActionArea>
+          </Card>
+          <Card
+            raised={paymentType === 'bankTransfer'}
+            sx={{
+              maxWidth: { sm: '100%', md: '50%' },
+              mb: 2,
+              flexGrow: 1,
+              outline: '1px solid',
+              outlineColor:
+                paymentType === 'bankTransfer' ? 'primary.main' : 'divider',
+              backgroundColor:
+                paymentType === 'bankTransfer' ? 'background.default' : '',
+            }}
+          >
+            <CardActionArea onClick={() => setPaymentType('bankTransfer')}>
+              <CardContent sx={{ display: 'flex', alignItems: 'center' }}>
+                <AccountBalanceRoundedIcon color="action" sx={{ marginRight: 2 }} />
+                <Typography variant="subtitle1">US bank account</Typography>
+              </CardContent>
+            </CardActionArea>
+          </Card>
+        </RadioGroup>
+      </FormControl>
+      {paymentType === 'creditCard' && (
+        <Box
+          sx={{
+            display: 'flex',
+            justifyContent: 'center',
+          }}
+        >
+          <Box
+            sx={{
+              display: 'flex',
+              flexDirection: 'column',
+              justifyContent: 'space-between',
+              p: 3,
+              height: { xs: '300px', sm: '350px' },
+              width: { xs: '450px', sm: '525px' },
+              borderRadius: '20px',
+              border: '2px solid ',
```

Comment:
```suggestion
              border: '1px solid ',
```

Much cleaner with a thiner border